creator-node/src/fileManager.js

```diff
@@ -688,5 +742,6 @@ module.exports = {
   checkFileMiddleware,
   getTmpTrackUploadArtifactsPathWithInputUUID,
   getTmpSegmentsPath,
-  copyMultihashToFs
+  copyMultihashToFs,
+  fetchFileFromNetworkAndWriteToDisk
```

Comment:
seems like only meant for internal consumption - lets not export until needed?

Reply:
yeah it's just hard to test fns without exporting those fns explicitly. I also dont think we should export fns that we only use internally but I feel like for the now it's okay \ wanted some test coverage on this specific flow

Reply:
oh right lol i forgot you used this in test ignore